pkgs/games/chiaki4deck/default.nix

```
@@ -54,6 +55,16 @@ mkDerivation rec {
     speexdsp
   ];
 
+  pythonPath = [
+    python3.pkgs.requests
+  ];
+
+  postInstall = ''
+    install -Dm755 $src/scripts/psn-account-id.py $out/bin/psn-account-id
+  '';
+
+  postFixup = "wrapPythonPrograms";
```

```suggestion
  postFixup = ''
    wrapPythonPrograms
  '';
```

To make sure it is a "script"

---

Done, thanks.